libs/api/src/services/scan/index.ts

```diff
@@ -274,6 +276,54 @@ export type DeletePrecinctSelectionConfigResponse = OkResponse;
  */
 export const DeletePrecinctSelectionConfigResponseSchema = OkResponseSchema;
 
+/**
+ * @url /config/polls
+ * @method GET
+ */
+export type GetPollsStateConfigResponse = OkResponse<{
+  pollsState: PollsState;
+}>;
+
+/**
+ * @url /config/polls
+ * @method GET
+ */
+export const GetPollsStateConfigResponseSchema: z.ZodSchema<GetPollsStateConfigResponse> =
+  z.object({
+    status: z.literal('ok'),
+    pollsState: PollsStateSchema,
+  });
+
+/**
+ * @url /config/polls
+ * @method PUT
+ */
+export interface PutPollsStateConfigRequest {
+  pollsState: PollsState;
+}
+
+/**
+ * @url /config/polls
+ * @method PUT
+ */
+export const PutPollsStateConfigRequestSchema: z.ZodSchema<PutPollsStateConfigRequest> =
+  z.object({
+    pollsState: PollsStateSchema,
+  });
+
+/**
+ * @url /config/polls
+ * @method PUT
+ */
+export type PutPollsStateConfigResponse = OkResponse | ErrorsResponse;
+
+/**
+ * @url /config/polls
+ * @method PUT
+ */
+export const PutPollsStateConfigResponseSchema: z.ZodSchema<PutPollsStateConfigResponse> =
+  z.union([OkResponseSchema, ErrorsResponseSchema]);
+
```

Comment:
Yes please, I'll have two orders of boilerplate.

Reply:
All the boilerplate you could want:
![image](https://user-images.githubusercontent.com/1938/198382921-6c831161-a0e4-452e-bb9f-86c7db26c30c.png)